Stackup-Rwanda/devrepublic-bn-frontend
# #170947528 Users should be able to  chat

#### What does this PR do?
- add a feature whereby users should be able to chat

#### Description of Task to be completed?
- add chat component
- add the route for the feature

#### How should this be manually tested?
- clone repo
- run `npm run start:dev`
- use the following URL: `http://localhost:3000/login` to login
- use the following URL: `http://localhost:3000/chat` to see online users and chat

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
- [#170947528](https://www.pivotaltracker.com/story/show/170947528)

#### Screenshots (if appropriate)

#### Questions:
